examples/spec/integration/schedule_child_workflow_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'workflows/schedule_child_workflow'
+require 'workflows/hello_world_workflow'
+
+describe ScheduleChildWorkflow, :integration do
+  let(:cron_schedule) { "*/6 * * * *" }
+
+  it 'schedules a child workflow with a given cron schedule' do
+    child_workflow_id = 'schedule_child_test_wf-' + SecureRandom.uuid
+    workflow_id, run_id = run_workflow(
+      described_class,
+      child_workflow_id,
+      cron_schedule,
+      options: {
+        timeouts: { execution: 10 }
+      }
+    )
+
+    # Giving the parent workflow sufficient time to run and schedule the child
+    sleep 2
+
+    history_response = fetch_history(child_workflow_id, nil)
+    expect(
+      history_response.history.events.first.workflow_execution_started_event_attributes.cron_schedule
+    ).to eq(cron_schedule)
```

Comment:
Not sure if there's a better way to prepare or assert this - maybe by fetching history for the parent workflow, if we're able to wait for a specific child workflow started event?